components/organisms/TopNav/top-nav.tsx

```diff
@@ -18,43 +33,30 @@ const TopNav: React.FC = () => {
   const router = useRouter();
 
   return (
-    <header className="top-nav-container w-full fixed top-0 left-0 flex justify-between items-center z-50 py-0.5 bg-light-slate-3 border-b">
-      <div className="container flex items-start justify-between px-2 mx-auto sm:items-center md:px-16">
-        <div className="flex flex-wrap items-center gap-3 md:gap-8">
-          <HeaderLogo withBg={false} textIsBlack />
-          <nav>
-            <ul className="flex flex-wrap w-full gap-3 mb-3 ml-2 md:gap-8 sm:m-0 sm:w-auto">
-              <li>
-                {!!user ? (
-                  <Link
-                    className={`text-sm ${getActiveStyle(router.asPath === "/hub/insights")}`}
-                    href={"/hub/insights"}
-                  >
-                    Insights
-                  </Link>
-                ) : (
-                  ""
-                )}
-              </li>
-              <li>
-                <Link
-                  className={`text-sm ${getActiveStyle(router.asPath === `/${userInterest}/dashboard/filter/recent`)}`}
-                  href={`/${userInterest}/dashboard/filter/recent`}
-                >
-                  Explore
-                </Link>
-              </li>
-              <li>
-                <Link className={`text-sm ${getActiveStyle(router.pathname === "/feed")}`} href={"/feed"}>
-                  Highlights
-                </Link>
-              </li>
-            </ul>
-          </nav>
-        </div>
-        <AuthSection />
-      </div>
-    </header>
+    <nav className={className}>
```

Comment:
Refactor (blocking): There are more than one `<nav />` elements on the page. To help screen readers, [add an aria-label for the nav element](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/navigation_role#associated_wai-aria_roles_states_and_properties). Insights seem to make sense for the name, but open to suggestions. See 

```suggestion
    <nav className={className} aria-label="Insights">
```

Reply:
Yea, its something around `main`